.openshift-ci/google-cloud-sdk/init.sh

```diff
@@ -1,15 +1,32 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-which gcloud
+# GCloud SDK initialization script could be invoked from two environment, Prow
+# or a GCP VM.
+
+which gcloud || true
 echo "Updating gcloud/gsutil ..."
+
+export CLOUDSDK_CONFIG=/tmp/gcloudconfig
+
+# Do not bother showing prompt e.g. for confirming deleting a VM
 gcloud config set core/disable_prompts True
-gcloud components install gsutil -q
-gcloud components update -q
+
+# If it happens on a GCP VM, gcloud is already there and don't have to be
+# updated manually (only via package manager)
```

Comment:
I don't fully understand this comment, are we simply stating the commands might fail when running on GCP?

Reply:
Nope, on GCP VM it fails updating saying it has to be done via package manager. Instead of doing it conditionally I just skip on fail here.